docs/src/templates/pages/documentation/general/gulp.hbs

```diff
@@ -1,169 +1,120 @@
 
+For a full list of tasks avilable, [take a look at the modules documentation](https://github.com/justeat/gulp-build-fozzie#the-gulp-tasks).
 
 
-<hr class="sectionSplitter">
-
-<a name="npm"></a>
-## NPM
-
-We use NPM – Node Package Manager – to install and manage the Gulp packages that we use.  NPM comes packaged with Node, so you don’t have to instal anything extra to use it.
-
-To see a full list of the packages we use, check out the [package.json](https://github.je-labs.com/I18N-Production-ROTW/ConsumerWeb/tree/master/src/ConsumerWeb/package.json) in Consumer Web.  When using Gulp for the first time, you will need to install these dependencies through Visual Studio, or by running the following command in a command prompt or terminal:
-
-```sh
-npm install
-```
-
----
-
-### NPM and Windows file paths
-
-When using NPM v2 on Windows, it can be hard to delete the `node_modules` folder, due to the [path names produced by project dependencies being too long](http://stackoverflow.com/questions/28175200/unable-to-delete-node-modules-folder).
-
-This will [soon be resolved](http://www.infoq.com/news/2015/06/npm) with the release of NPM v3, which is currently in beta, but until then, there is a workaround should you ever need to do this.
+<a name="gulp-vs"></a>
+### Using Gulp in Visual Studio
 
-Install a package called **rimraf** by running `npm install rimraf -g` from the command line.  Then, from your project folder, you can delete your `node_modules` folder by typing `rimraf node_modules`.
+Here we document the steps needed to successfully run Gulp using Visual Studio 2017.
 
+#### Setup steps
 
-<hr class="sectionSplitter">
+To get up and running with using Gulp in your setup of Visual Studio, just carry out the following steps:
 
-<a name="gulp-tasks"></a>
-## Gulp Tasks
+1. **[Install Node.js](https://nodejs.org/)**
 
-All tasks can be run from Visual Studio by right clicking the name of the task, and selecting **Run**.
+	Node.js – and it’s bundled package manager npm – are needed to run Gulp tasks with VS 2013 or VS 2015 Professional.
 
----
+	Installing this should be as simple as going to the [Node.js website](https://nodejs.org/) and hitting the 'Install' button, which will take you through the install process.  Go with all the defaults throughout the install process and this should install Node and it’s package manager on your system.
 
-### gulp default
+2. **Install Gulp**
 
-The default Gulp task is made up of 3 other tasks.  It will run the [css](#gulp-css), [scripts](#gulp-scripts) and [copy:fonts](#gulp-copy) Gulp tasks, which will give you a compiled version of the front–end code for use in your local build.
+	Open up a command prompt or terminal window and enter `npm install gulp -g`.  This should install Gulp so that Visual Studio can do it’s magic.
```

Comment:
it's -> its

Reply:
I clearly need training on the use of its vs it’s 😉